specifications/api/swagger.yaml

```diff
@@ -1322,9 +1322,11 @@ components:
       required:
         - estimated_min
         - estimated_max
+        - deposit
       properties:
         estimated_min: *transactionAmount
         estimated_max: *transactionAmount
+        deposit: *transactionAmount
```

Comment:
For consistency, what about having a list of `deposits` here too ?

Reply:
Is it possible to have theoretically multiple for a fee estimation?

Reply:
Theoretically yes. If we were to register multiple stake keys. Arguably, you could represent them as "one big deposit", but we're already taking the approach of showing them as multiple deposits in transactions.